source/agora/test/Ledger.d

```diff
@@ -132,46 +132,37 @@ unittest
     // transactions are ordered lexicographically by hash in the Merkle tree
     hashes.sort!("a < b");
 
-    const Hash ha = hashes[0];
```

Comment:
I imagined a tree like this.
```
               abcdefgh
        +---------++---------+
      abcd                  efgh
   +---++---+            +---++---+
  ab        cd          ef        gh
+-++-+    +-++-+      +-++-+    +-++-+  
a    b    c    d      e    f    g    h
```

Reply:
I think the mapping between characters [a .. z] is simple to [0 .. $]. He only replaced the bottom level.

Reply:
Yeah, I got it.